plugins/generator-1.20.4/neoforge-1.20.4/mappings/screens.yaml

```diff
@@ -2,96 +2,103 @@ _default: Ingame
-Enchantment: EnchantmentScreen
+EffectRenderingInventory: EffectRenderingInventoryScreen
 Error: ErrorScreen
+Experiments: ExperimentsScreen
 FlatPresets: PresetFlatWorldScreen
 GamemodeSelection: GameModeSwitcherScreen
+GenericWaiting: GenericWaitingScreen
 GPUWarning: PopupScreen
-Grindstone: GrindstoneScreen
-Hopper: HopperScreen
-Jigsaw: JigsawBlockEditScreen
+Inventory: InventoryScreen
+ItemCombiner: ItemCombinerScreen
+KeyBinds: KeyBindsScreen
 Language: LanguageSelectScreen
-Lectern: LecternScreen
-Loom: LoomScreen
 MemoryError: OutOfMemoryScreen
 MouseSettings: MouseSettingsScreen
 Multiplayer: JoinMultiplayerScreen
 MultiplayerWarning: SafetyScreen
+NameReport: NameReportScreen
+OnlineOptions: OnlineOptionsScreen
 OptimizeWorld: OptimizeWorldScreen
 Options: OptionsScreen
 OptionsSounds: SoundOptionsScreen
 Pack: PackSelectionScreen
-ReadBook: BookViewScreen
+RecoverWorldData: RecoverWorldDataScreen
+ReportPlayer: ReportPlayerScreen
+ReportReasonSelection: ReportReasonSelectionScreen
 ServerList: DirectJoinServerScreen
+ServerReconfig: ServerReconfigScreen
 Settings: OptionsSubScreen
 ShareToLan: ShareToLanScreen
+SkinReport: SkinReportScreen
 SleepInMultiplayer: InBedChatScreen
+SocialInteractions: SocialInteractionsScreen
 Stats: StatsScreen
+SymlinkWarning: NoticeWithLinkScreen
+TelemetryInfo: TelemetryInfoScreen
+UnsupportedGraphicsWarning: UnsupportedGraphicsWarningScreen
 VideoSettings: VideoSettingsScreen
+Warning: WarningScreen
+Win: WinScreen
 WithNarratorSettings: SimpleOptionsSubScreen
 Working: ProgressScreen
 WorldLoadProgress: LevelLoadingScreen
-WorldSelection: SelectWorldScreen
-Anvil: AnvilScreen
-Beacon: BeaconScreen
-BlastFurnace: BlastFurnaceScreen
-BrewingStand: BrewingStandScreen
-CartographyTable: CartographyTableScreen
-Chest: ContainerScreen
-Crafting: CraftingScreen
-Creative: CreativeModeInventoryScreen
-Dispenser: DispenserScreen
-Furnace: FurnaceScreen
-HorseInventory: HorseInventoryScreen
-Inventory: InventoryScreen
-Merchant: MerchantScreen
-ShulkerBox: ShulkerBoxScreen
-SmithingTable: SmithingScreen
-Smoker: SmokerScreen
-Stonecutter: StonecutterScreen
-KeyBinds: KeyBindsScreen
-AccessibilityOnboarding: AccessibilityOnboardingScreen
-AccessibilityOptions: AccessibilityOptionsScreen
-CreditsAndAttribution: CreditsAndAttributionScreen
-GenericWaiting: GenericWaitingScreen
-OnlineOptions: OnlineOptionsScreen
-SymlinkWarning: SymlinkWarningScreen
-Win: WinScreen
-Advancements: AdvancementsScreen
-EffectRenderingInventory: EffectRenderingInventoryScreen
-HangingSignEdit: HangingSignEditScreen
-ItemCombiner: ItemCombinerScreen
-Warning: WarningScreen
-ChatReport: ChatReportScreen
-ChatSelection: ChatSelectionScreen
-ReportReasonSelection: ReportReasonSelectionScreen
-SocialInteractions: SocialInteractionsScreen
-TelemetryInfo: TelemetryInfoScreen
-ConfirmExperimentalFeatures: ConfirmExperimentalFeaturesScreen
-Experiments: ExperimentsScreen
\ No newline at end of file
+WorldSelection: SelectWorldScreen
```

Comment:
`Realms32bitWarningScreen` appears to be missing. Since we already have some abstract classes (albeit, without "Abstract" in their names), we should probably also add the following: `AbstractCommandBlockEditScreen`, `AbstractContainerScreen`, `AbstractFurnaceScreen`, `AbstractSignEditScreen`, `AbstractReportScreen`. This obviously also applies to the screens datalist.

Reply:
> All other screens (not related to Realms)

Adding this screen would probably lead to adding more of Realms ones.

Reply:
This is the only one. Matej ran it through porting tools

Reply:
(https://github.com/MCreator/DevTools)

Reply:
I'm not sure why this screen is defined in one of main MC packages and the rest of them are located in `com.mojang.<...>` package.

Reply:
I've re-run the checker on the entire source and found a few more that could be added, namely:

- DisconnectedRealmsScreen
- RealmsScreen

Since this isn't a lot of screens outside the "screens" package, I think we should consider adding them all for the sake of completion.

Reply:
From what I remember, screens in `com.mojang.<...>` package extend one of these (the other one does too). So related to Realms, but we'd need to add other screens of the kind as well then I'd say.

Reply:
I don't think that is needed. The core net.minecraft package is bound to the Minecraft version, while the realms part can be updated separately by the launcher.